net/socket/send.c

```diff
@@ -156,30 +156,7 @@ ssize_t psock_send(FAR struct socket *psock, FAR const void *buf, size_t len,
 
 ssize_t send(int sockfd, FAR const void *buf, size_t len, int flags)
```

Comment:
You mean remove the system call for send() ? I guess it could be done but I think we should keep it as a system call.

Reply:
> You mean remove the system call for send() ?

Yes.

> I guess it could be done but I think we should keep it as a system call.

Normally, if the function can be simply forwarded to system call, it should be move to libs/libc as normal function.

Reply:
Ok, what about recv() then, should that be removed too ? Maybe we can do another patch for that because it is not related to this fix ?

Reply:
could be.